modules/coreos-layering-configuring.adoc

```diff
@@ -30,7 +30,7 @@ For example, the following Containerfile creates a custom layered image from an
 [source,yaml]
 ----
 # Using a 4.12.0 image
```

Comment:
For consistency and clarity, We may want to make this comment something like "# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`"